docs/testcases/TC06-008-fail.html

```diff
@@ -136,7 +136,7 @@ <h4 id="TCcode">Code</h4>
   								<h4 id="TCurl">Single-Page URL</h4>
   									<p><a href="../testfiles/TF06/06-008.html">https://section508coordinators.github.io/BaselineTestPages/testfiles/TF06/06-008.html</a></p>
   								<h4 id="TCframe">Embedded Code</h4>
-  									<iframe src="../testfiles/TF06/06-008.html" class="card shadow my-3"></iframe>
+  									<iframe title="embedded test file" src="../testfiles/TF06/06-008.html" class="card shadow my-3"></iframe>
   						<h2 id="TCsteps">Test Steps</h2>
   							<table class="table table-hover">
   								<thead class="thead-dark">
```

Comment:
TC06-008-fail.html:

  - Typo: Minor issue: "Hwoever" instead of "However" under heading "Test Case Description"
  - Minor issue: The code `<p>` value in the test case does not match the actual `<p>` value in the embedded test file. The difference is minor but different.

Reply:
Fixed